pkgs/tools/X11/xosview2/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, xorg }:
```

Comment:
Better be more specific and change ```xorg``` to ```libX11``` as well as in the ```buildInputs```. Otherwise looking very good!